src/popup/components/ShareAddress.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <ShareAddressBase
+    v-bind="$attrs"
```

Comment:
I suggest to avoid passing all props to child components this way because we are losing typings. Maybe you can follow the same pattern used in `src/protocols/ethereum/views/TransferSendModal.vue`?